src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultPublisher.cs

```diff
@@ -1,55 +1,15 @@
 namespace NServiceBus.AcceptanceTests.EndpointTemplates
 {
     using System;
-    using System.Collections.Generic;
-    using System.Linq;
     using System.Threading.Tasks;
-    using NServiceBus.AcceptanceTesting;
     using NServiceBus.AcceptanceTesting.Support;
     using NServiceBus.Config.ConfigurationSource;
-    using NServiceBus.Pipeline;
-    using NServiceBus.Pipeline.Contexts;
-    using NServiceBus.Routing.StorageDrivenPublishing;
 
     public class DefaultPublisher : IEndpointSetupTemplate
     {
         public Task<BusConfiguration> GetConfiguration(RunDescriptor runDescriptor, EndpointConfiguration endpointConfiguration, IConfigurationSource configSource, Action<BusConfiguration> configurationBuilderCustomization)
         {
-            return new DefaultServer(new List<Type> { typeof(SubscriptionTracer), typeof(SubscriptionTracer.Registration) }).GetConfiguration(runDescriptor, endpointConfiguration, configSource, b =>
```

Comment:
Misstake, I need to sync with @SzymonPobiega since this will be trickier to support with the new batching. I'll re-add it in some form